cmd/thor/flags.go

```diff
@@ -183,6 +183,12 @@ var (
 		Usage: "comma separated list of allowed API tracers(none,all,call,prestate etc.)",
 	}
 
+	minMaxPriorityFeePerGasFlag = cli.Uint64Flag{
+		Name:  "minimum-max-priority-fee-per-gas",
```

Comment:
Should be `minimum-priority-fee-per-gas`, we are verifying the 'acting' priority fee per gas here.

Reply:
The name of the attribute in the transaction is `maxPriorityFeePerGas`, I think the current name makes more sense since that is what we are validating.

Reply:
I think the feature should be validating effective priority fee per gas other than max priority fee per gas, since `effectivePriorityFeePerGas = min(tx.maxPriorityFeePerGas, tx.maxFeePerGas - block.baseFeePerGas)`

Reply:
If that is the case, does this comment still apply as is? https://github.com/vechain/protocol-board-repo/issues/413#issuecomment-2705738256

Reply:
Done assuming this